source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -3,278 +3,528 @@
+   located in :file:`/home/clr/mix/update/www/{<MIXVERSION>}`, where
+   ``<MIXVERSION>`` is the mix version you defined (10 by default).
+
+   By default, mixer will use the legacy swupd-server to generate the update
+   content. Mixer now has a new implementation built into the mixer tool itself.
+   While this is currently an experimental feature, you can (and should) use the
+   new swupd-server by passing the :option:`--new-swupd` flag. The legacy
+   swupd-server will soon be deprecated, and mixer will use the new version
+   automatically.
 
    All content to make a fully usable mix will be created by this step, but
-   note that only zero packs are automatically generated. To create optional
-   delta packs, run the pack-maker as follows::
+   note that only *zero packs* are automatically generated. Zero packs are
+   the content needed to go from nothing to the mix version you just built
+   content for. To create optional *delta packs*, which allow for
+   transitioning from one mix version to another, run the pack-maker as
+   follows:
+
+   .. code-block:: bash
 
       # sudo mixer-pack-maker.sh --to <MIX_VERSION> --from <PAST_VERSION> -S /home/clr/mix/update
 
-   The pack-maker will generate all delta packs for changed bundles from
-   PAST_VERSION to MIX_VERSION. If your STATE_DIR is in a different location be
-   sure to specify where with the -S option. For the first build, no delta
-   packs can be created because the "update" is from version 0, which impicitly
-   has no content, thus no deltas can be generated. For subsequent builds,
-   mixer-pack-maker.sh can be run to generate delta content between them (i.e
-   10 to 20).
+   The pack-maker will generate all delta packs for bundles that have changed
+   from ``PAST_VERSION`` to ``MIX_VERSION``. If your ``STATE_DIR`` is in a
+   different location, be sure to specify the location with the ``-S``
+   option. For the first build, no delta packs can be created because the
+   "update" is from version 0. Version 0 impicitly has no content, thus no
+   deltas can be generated. For subsequent builds,
+   :file:`mixer-pack-maker.sh` can be run to generate delta content between
+   them (for example: 10 to 20).
 
-#. **Creating an image** To create a bootable image from your update content,
-   you will need the configuration file for ister to create images::
+#. **Creating an image**. Mixer uses the ``ister`` tool to create a bootable
+   image from your updated content. To configure the image ``ister`` creates,
+   you will need the ``ister`` config file. You can obtain a default value
+   from the ``ister`` package:
 
-      # curl -O https://raw.githubusercontent.com/bryteise/ister/master/release-image-config.json
+   .. code-block:: bash
 
-   Edit this to include  all the bundles you want pre-installed into your
-   image. For a minimal, base image this would be::
+      # cp /usr/share/defaults/ister/ister.json relase-image-config.json
```

Comment:
@kevincwells, I think "/usr/share/defaults/ister/ister.json" is not a good example for users for a number of reasons:
- "disk:" is set to "/dev/sdb"
- "Bundles:" doesn't show the minimal bundles
- "DestinationType" is set to "Physical"  

I would prefer to keep using https://raw.githubusercontent.com/bryteise/ister/master/release-image-config.json or modify "/usr/share/defaults/ister/ister.json" to match.  

I'm just thinking it's good to make the example simple and the way we intend it to be, especially for new users.

Reply:
I don't think we can say what way "we intend it to be" -- the whole point of `ister`'s config is that it can be used to build for any number of different formats, and those will be entirely dependent on the user. The `release-image-config.json` you mention (which we still link to in the sentence immediately following the lines mentioned here) is not necessarily representative of the image format any given customer will want. The point of this command is to give them a starting point for the syntax.

I'm open to changing the default config that is shipped with `ister` (though that's a different project and would need to be made there), but we deliberately moved away from telling the user to curl a file from github for this step.